src/wikibus.sources.EF/SourcesRepository.cs

```diff
@@ -44,7 +48,7 @@ public Magazine GetMagazine(Uri identifier)
         [return: AllowNull]
         public Brochure GetBrochure(Uri identifier)
         {
-            var id = this.identifierRetriever.GetBrochureId(identifier);
+            var id = IdentifierOf<Brochure>.Match(identifier, this.config.BaseResourceNamespace).Get<int?>("id");
```

Comment:
Same here with the base. If the `identifier` is a complete URI it is possible to add it. Only needed when the actual template is a just the path